Content.Server/Psionics/PsionicsSystem.cs

```diff
@@ -76,13 +73,16 @@ private void OnMeleeHit(EntityUid uid, AntiPsionicWeaponComponent component, Mel
                     return;
                 }
 
-                if (component.Punish && HasComp<PotentialPsionicComponent>(entity) && !HasComp<PsionicComponent>(entity) && _random.Prob(0.5f))
+                if (component.Punish && !HasComp<PsionicComponent>(entity) && _random.Prob(0.5f))
                     _electrocutionSystem.TryDoElectrocution(args.User, null, 20, TimeSpan.FromSeconds(5), false);
             }
         }
 
-        private void OnInit(EntityUid uid, PsionicComponent component, ComponentInit args)
+        private void OnInit(EntityUid uid, PsionicComponent component, ComponentStartup args)
         {
+            component.AmplificationSources.Add("Baseline Amplification", _random.NextFloat(0.4f, 1.2f));
+            component.DampeningSources.Add("Baseline Dampening", _random.NextFloat(0.4f, 1.2f));
```

Comment:
Like with https://github.com/Simple-Station/Einstein-Engines/pull/731/files#r1718973998, it may be a good idea to turn these floats into consts to make them easier to find